### DISCUSSION TITLE
Game bot to do automation tasks.  Must be able to do 50 invites be able to change out invite code.  Be able to run on a schedule if need be. I will send written and video of the process it needs to do.

### DISCUSSION THREAD

**Discussion by dollarotieno** [community/community#66407] [New to GitHub]
I will attach the video and write out method plus tasks I need done for this to be an automation bot basically. 
Please ignore the reference to tiny task in video as it is not efficient at all on my pc for some reason. 
I have a rog "zep" i9 64 gb 4070 
Gaming laptop/pc. 
So it can handle quite a bit. 

What I need is to be able to swap invite links out manually to bot and for bot to run a course of 50 invites through the method video I am sending. 
I also need to be able to set it to run on a schedule on its own that I can set for either 1hr or over night if need be. 
You will have to show me as well as write out how to operate bot please.

**Comment by queenofcorgis** [comment]
Hi @dollarotieno , we're happy you're here 😀

However, we currently do not allow job listings or sourcing folks to code for you on the Community on Discussions. It is something that we've added to the list to investigate, if we can do so without overwhelming other conversations. Keep checking back as the Community grows and we add more capabilities!

Thanks for your interest 👍
